Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue2680ScrollView.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Linq;
+using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+#if UITEST
+using Xamarin.UITest; 
+using NUnit.Framework; 
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 2680, "[Enhancement] Add VerticalScrollMode/HorizontalScrollMode to ListView and ScrollView", PlatformAffected.All)]
+	public class Issue2680ScrollView : TestContentPage // or TestMasterDetailPage, etc ... 
+	{
+		protected override void Init()
+		{
+			// Initialize ui here instead of ctor 
+			var longStackLayout = new StackLayout();
+			Enumerable.Range(1, 50).Select(i => new Label() { Text = $"Test label {i}" })
+				.ForEach(label => longStackLayout.Children.Add(label));
+
+			var scrollView = new ScrollView
+			{
+				Orientation = ScrollOrientation.Vertical,
+				Content = longStackLayout
+			};
+
+			Content = scrollView;
+
+			scrollView.Orientation = ScrollOrientation.Neither;
+		}
+
+#if UITEST
+		[Test] 
+		public void Issue2680Test() 
+		{ 
+			RunningApp.Screenshot("I am at Issue 2680"); 
+			RunningApp.ScrollDown(); 
+			RunningApp.ScrollDown(); 
+			RunningApp.ScrollDown(); 
+			RunningApp.WaitForElement(q => q.Marked("Test label 1")); 
+			RunningApp.Screenshot("I'm still seeing the first Label in non-scrolable ScrollView"); 
```

Comment:
non-scrollable >> non-scrollable

Reply:
Fixed in https://github.com/xamarin/Xamarin.Forms/pull/3113/commits/9538045d025c61f6861c89caba30ae4c275707eb